index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>D.O.A.</title>
+<!-- fonts here -->
+	<link rel="stylesheet" href="">
+</head>
+
+<body>
+
+
+<!-- HEADER -->
+
+	<header id="header" class="">
+		<nav>
+			<h1 id="navHome"><a href="index.html" title="Home">D.O.A.</a></h1>
+			<div class="navLinks"><a href="products.html" title="Our_Bikes">Our Bikes</a></div>
+		</nav>
+	</header>
+
+
+<!-- HOMEPAGE CONTENT -->
+
+	<div class="mainContent">
+		<div id="splashPicture">
+			<img src="images/bikeRide.jpg" alt="Vroom.">
```

Comment:
Whoops, I forgot to mention that here and in contact.html, the images are in no way real. Just placeholders.